src/xdocs/property_types.xml

```diff
@@ -30,21 +30,28 @@
     </p>
     </section>
 
-    <section name="integer">
+    <section name="Integer">
       <p>
         This type represents an integer. The string representation is
         parsed using the <code>java.lang.Integer</code> class.
       </p>
     </section>
 
-    <section name="string">
+    <section name="Double">
+      <p>
+        This type represents an integer. The string representation is
```

Comment:
Double is not an integer. Integers have no decimal points, doubles do.

Reply:
Sorry for that, forgot to change after copying the Integer description